content/articles/road-layout-design-programatically-using-matlab/index.md

```diff
@@ -0,0 +1,177 @@
+
+### Road layout design programmatacally using matlab
+### Table of content
+- [Road layout design programmatacally using matlab](#road-layout-design-programmatacally-using-matlab)
+- [Table of content](#table-of-content)
+- [Introduction](#introduction)
+- [### Prerequisites](#-prerequisites)
+- [Straight road](#straight-road)
+- [Roads with lanes](#roads-with-lanes)
+- [Road intersection](#road-intersection)
+- [Curved roads layout](#curved-roads-layout)
+- [Roundabout road layout](#roundabout-road-layout)
+- [Elevated road layout](#elevated-road-layout)
+- [Overpass roads](#overpass-roads)
+- [Conclusion](#conclusion)
+
+
+
+### Introduction
+In designing road layout, we use the `drivingscenario` function. `scenario` function allows for simulation of an activity or a structure that can be found in the real world. `Drivingscenario` function can be used to create different designs of road layout by plotting the road layout in 2D or 3d graphs. This is done by specifying the coordinates of the road center using function `roadCentre=[];` and the road width then plotting these points on the graph.
+
+This article will discuss the design of the  following road layout using Matlab functions:
+
+- Straight road.
+- Roads with lanes.
+- Intersecting roads.
+- Curved road layouts.
+- Roundabout layout.
+- Elevated road layout.
+- Overpass road layout.
+
+### ### Prerequisites
+To follow along with this tutorial, you'll need:
+- [MATLAB](https://www.mathworks.com/products/get-matlab.html?s_tid=gn_getml) installed.
+- [Proper understanding](/engineering-education/getting-started-with-matlab/) of MATLAB basics.
+
+### Straight road
+A straight road layout has a fixed width. It can be designed by defining the road center coordinates with two points and a specified width. The graph axis is labeled in meters.
+
+The first step of designing any road layout is defining the `Scenario` function as  `drivingscenario`, that is `scenario=drivingScenario`.  After the declaration of scenario function the road center coordinates are specified for our case we will use (0 10) and (60 10) with a road width of 8 meters. The final road layout is gotten by plotting the `scenario`, road centerline, and width. The resultant figure will have a straight road layout.
```

Comment:
This paragraph is quite long, consider breaking it down. You may also rewrite this part: `.... function as  `drivingscenario`, that is `scenario=drivingScenario...`

Reply:
> This paragraph is quite long, consider breaking it down. You may also rewrite this part: `.... function as `drivingscenario`, that is `scenario=drivingScenario...`

Work on punctuations as well to make your sentences easy to read!

Reply:
Simplify your words, for example, `obtained` is better than `gotten`. Be as simple as possible!